doc/examples/scripts/sequence/plotepiscan.py

```diff
@@ -0,0 +1,407 @@
+                            c=c+1
+                    elif c % 2 != 0: 
+                        if a==None:
+                            gapped.insert(x,(template[x],0)) 
+                            x=x+1
+                        elif a != lk1[b]:
+                            gapped.insert(x,(template[x],0))         
+                            x=x+1
+                            c=c+1
+                        else:        
+                            x=x+1
+                            c=c+1
+                            break
+
+    # For terminal gaps
+    if len(gapped) < len(template) and template[len(gapped)+1]== None:            
+        gapped_tail=[]
+        for n in range(len(template)-len(gapped)):
+            gapped_tail.append(('None', 0))                
+        gapped = gapped + gapped_tail
+   
+    return gapped
+
+# Lets build the list of tuples with gapped sequences
+# FCR3 array, overlap_step: 1 (pep = 20-mer with 19 overlap)
+gapd_s1 = gapped_seq(dfa, traceA, 20, 1)
+
+# NF54 array, overlap_step: 2 (pep = 20-mer with 18 overlap)
+gapd_s2 = gapped_seq(dfb, traceB, 20, 2) 
+
+# Checkpoint. Both lists of tuples must have the same length.
+len(gapd_s1) == len(gapd_s2)
+
+########################################################################
+# Create a signal map
+# -------------------
+
+# Here we build a numpy.ndarray to map the signal scores and corresponding 
+# position of the score residue on the sequence alignment.
+def signal_map(gapped_seq1, gapped_seq2,):
+    gapd_s1 = gapped_seq1
+    gapd_s2 = gapped_seq2
+    fl_score = np.zeros((len(gapd_s1),2))
+    
+    for v1 in range(len(gapd_s1)):
+        fl_score[v1,0] = gapd_s1[v1][1]    
+        fl_score[v1,1] = gapd_s2[v1][1]
+        
+    return fl_score
+
+
+score = signal_map(gapd_s1, gapd_s2)
+
+########################################################################
+# Sequence alignment decorated with MAb recognition regions 
+# ---------------------------------------------------------
+#
+# Now we can plot the sequence alignment using an :class:`ArrayPloter` 
+# instance that higlights sequence recognition regions at the positions 
+# of the respective score residue per alignment column.
+# To easily interpret the intensity-decorated alignment we will add a 
+# colorbar scaled accordingly. The scale matches the transformation 
+# applied to the recognition signal recorded on the score ndarray.
+
+import biotite.sequence.graphics as graphics
+
+fig = plt.figure(figsize=(20, 16))
+ax = fig.add_subplot(111)
+graphics.plot_alignment_array(
+    ax, alignments[0], fl_score = score, labels = ["FCR3", "NF54"],
+    show_numbers = True, symbols_per_line = 120,
+    show_line_position = True) 
+
+# Add the axes where the colorbar will reside:
+ax2 = fig.add_axes([0.1,-0.005, 0.8, 0.03])
+ax2.set_frame_on(False)
+
+# Access the colormap of the relevant instace of ArrayPlotter: 
+colormap = graphics.ArrayPlotter(ax2, score).get_cmap()
```

Comment:
I rendered the example for the gallery, and the output image is shown relatively small.
Could you use a width of 8.0 used also by the other examples and adapt symbols per line, symbols size and plot height accordingly? An example of good parameter values can be seed in https://www.biotite-python.org/examples/gallery/sequence/pi3k_alignment.html.

Furthermore, the colorbar is missing in this plot. I think you have to combine this code block with the one below to achieve what you want to show

Reply:
Thank you. I adjusted the parameters following the example provided. It should be correct now, the colorbar is also fixed and showing up.

Reply:
Thanks. However, when I render the gallery the colorbar has no axis label. Can you confirm this observation:

![image](https://github.com/biotite-dev/biotite/assets/28051833/26a9699d-12f8-4e52-b016-5763cc60a480)

Reply:
Hi, I fixed the rendering. Thanks
  
![plot](https://github.com/biotite-dev/biotite/assets/123925494/ae04de24-2445-4d2f-96c5-f311ae19eb18)